hybridse/src/vm/transform.cc

```diff
@@ -2179,5 +2179,9 @@ void RequestModeTransformer::ApplyPasses(PhysicalOpNode* node,
     return;
 }
 
+Status RequestModeTransformer::TransformLoadDataOp(const node::LoadDataPlanNode* node, PhysicalOpNode** output){
```

Comment:
Add a unit test in transform_test.cc or transform_request_mode_test.cc and checks the error message.

Reply:
ok, I'll add it in transform_request_mode_test.cc